con-master/src/main/java/ba/codecentric/master/controller/HotelController.java

```diff
@@ -19,7 +23,15 @@ public HotelController(HotelService hotelService) {
     }
 
     @RequestMapping(value = "/hotel/save", method = RequestMethod.POST)
-    public void saveHotel(@RequestBody Hotel hotel) {
-        hotelService.saveHotel(hotel);
+    public ResponseEntity<Hotel> saveHotel(@RequestBody Hotel hotel) {
+        return new ResponseEntity<Hotel>(hotelService.saveHotel(hotel), HttpStatus.OK);
+
+    }
+
+    @RequestMapping(value = "/hotel/get", method = RequestMethod.GET)
+    public ResponseEntity<Hotel> getHotel() {
+        List<Hotel> hotels = hotelService.getHotels();
```

Comment:
Why do you use "select all" to fetch a single hotel? You can use SELECT 1 instead.